extension/js/common/org-rules.ts

```diff
@@ -172,7 +172,8 @@ export class OrgRules {
    * This is because they already have other means to obtain public keys for these domains, such as from their own internal keyserver
    */
   public canLookupThisRecipientOnAttester = (emailAddr: string): boolean => {
-    return !(this.domainRules.disallow_attester_search_for_domains || []).includes(Str.getDomainFromEmailAddress(emailAddr) || 'NONE');
+    const forbidden = ['*', Str.getDomainFromEmailAddress(emailAddr) || 'NONE'];
```

Comment:
I know it's a nitpick but could we dumb down the code a bit? :-) The original code was already not ideal but ok for the simple functionality, but now it's getting a bit harder for me to follow. For example:

```ts
  public canLookupThisRecipientOnAttester = (emailAddr: string): boolean => {
    const disallowedDomains = this.domainRules.disallow_attester_search_for_domains || [];
    if(disallowedDomains).includes('*')) {
      return false;
    }
    const userDomain = Str.getDomainFromEmailAddress(emailAddr);
    if(!userDomain) {
      ??? I wonder if we should throw here?
    }
    return !(disallowedDomains).includes(userDomain);
  }
```

It's more verbose, but easier to follow.

Reply:
Done